swagger.yaml

```diff
@@ -1536,6 +1536,9 @@ components:
           format: date
           type: string
           description: Original end date of the milestone
+        estimatedReadiness:
+          type: string
```

Comment:
Is it really just a string? Not a date?

Reply:
<img width="1053" alt="Screenshot 2024-05-27 at 9 51 20" src="https://github.com/Metatavu/lipsanen-project-management-spec/assets/22254327/c9de63b8-c472-465a-8cb3-e2a0e8e9e548">
It is that percentage value that is hand defined by the Admin / Project owner

Reply:
The same schema parameter in Task had string, so I assumed it is not different for the Milestone

Reply:
I have set both milestone and task estimated readiness as a number. There was some mention yesterday of it also being a float? Is this necessary, I would think a number 1 - 100 is suitable here?

Reply:
It probably depends just on the UI requirements, float was mentioned by @Daniilmass

Reply:
After discussion estimatedReadiness as integer is fine.